src/pages/Homepage/TopMessageContentful.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+	<div v-if="content" class="homepage-top-message">
+		<div class="row align-center">
+			<div class="columns small-12" v-html="content">
+			</div>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		content: {
+			type: String,
+			default: '',
+		},
+	},
+};
+</script>
+
+<style lang="scss">
```

Comment:
the styles don't get applied to the inner content because it does not receive the scope attribute. I could use the deep styling selector, but I could also just use a unique class name, which I chose to do here using "homepage-top-message"